code/game/machinery/computer/_computer.dm

```diff
@@ -67,7 +67,7 @@
 /obj/machinery/computer/play_attack_sound(damage_amount, damage_type = BRUTE, damage_flag = 0)
 	switch(damage_type)
 		if(BRUTE)
-			if(machine_stat & BROKEN)
+			if(damage_amount)
```

Comment:
Why this change?

Reply:
thaat change shouldnt be in there, i was looking at that code as reference, must have typed there by accident. oops.